halo2_debug/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+use rand_core::block::BlockRng;
+use rand_core::block::BlockRngCore;
+use rand_core::OsRng;
+use tiny_keccak::Hasher;
+
+// One number generator, that can be used as a deterministic Rng, outputing fixed values.
```

Comment:
BTW, the comments you added could be done with `///` and then they become documentation.